### ISSUE TITLE
APK files

### ISSUE THREAD

**Issue by IzzySoft** [jasoncheng/mawa-android#156]
May I suggest you attach the APK files to your [releases](https://github.com/jasoncheng/dica/releases) instead of storing them in a separate branch? Keeps the repo more compact and makes it easier to find them (plus my updater has it easier to fetch them (see #98); Github API has its limitations, unfortunately).

In case you are unsure about how to do that, please see [Creating releases](https://help.github.com/en/articles/creating-releases) in the Github documentation.

**Comment by IzzySoft** [comment]
Any chance, @jasoncheng? Are you still maintaining the app (not much activity here lately)?

**Comment by IzzySoft** [comment]
@jasoncheng ^^ ping – are you still there (and hopefully doing fine)?

**Comment by jasoncheng** [comment]
> @jasoncheng ^^ ping – are you still there (and hopefully doing fine)?

Hi, long time no see, sorry for the late to reply;
I have been busy on my job and other projects; 
How are you ? how about Friendica API progress?
I think i almost forgot everything on this project, haha.....
Recently i wrote lot of code on flutter, coding as daily routine :),
And soon, i think maybe my career will have some big change,  
I might rewrite this project w/ new language, open source, and support all platform(Web/App/Desktop App...), but have to wait more stable on my job....maybe few months later.

**Comment by IzzySoft** [comment]
Yeah, has been some time… I'm not on Friendica and not a dev, so I cannot tell about the API. But I'm glad for everything that makes the Fediverse grow :smile: In a case of emergency, even when it's written in Flutter or RN (where a simple "Hello world" results in a 5 MB `*.apk`) :speak_no_evil:

Best wishes for your carrier and a stable job then (first things first, if not necessarily in that order :rofl:) – and look forward to the rewrite!

**Comment by IzzySoft** [comment]
Hi @jasoncheng just a friendly ping: any news? Or shall I rather mark the app "currently unmaintained" and wait until you possibly find time for it? No bad feelings, just wanting to keep things transparent for the users. And it makes no sense to permanently annoy you with my pings if you cannot work on it anyway.

**Comment by IzzySoft** [comment]
Another half year, another friendly ping: Will you pick up again, or did you give up, @jasoncheng ?

**Comment by IzzySoft** [comment]
Yikes, more than a year again. I don't even know if that last release is still working, and no life signs from you, @jasoncheng :sob: Guess I'll have to remove your app from my repo for now and wait for your ping should you pick up again? Or does it really make sense to keep the old version up and wait?
